rtrlib/aspa/aspa_array/aspa_array.c

```diff
@@ -145,13 +145,13 @@ size_t aspa_array_search(struct aspa_array *vector, uint32_t customer_asn)
 	}
 
 	// left and right bound of our search space
-	size_t left = 0;
```

Comment:
as the overflow only occured on `right = center-1`, can we stick to `size_t` types and handle the extra case in the loop?
```c
//...
} else if (center_value > customer_asn) {
    if (right == 0)
        return -1;
    // value should be on the left side
    right = center - 1;
//...
```